src/app/organizations/vault/vault.component.ts

```diff
@@ -272,15 +272,15 @@ export class VaultComponent implements OnInit, OnDestroy {
 
         childComponent.organization = this.organization;
         childComponent.cipherId = cipher == null ? null : cipher.id;
-        childComponent.onSavedCipher.subscribe(async (c: CipherView) => {
+        childComponent.onSavedCipher.subscribe(async () => {
             this.modal.close();
             await this.ciphersComponent.refresh();
         });
-        childComponent.onDeletedCipher.subscribe(async (c: CipherView) => {
+        childComponent.onDeletedCipher.subscribe(async () => {
             this.modal.close();
             await this.ciphersComponent.refresh();
         });
-        childComponent.onRestoredCipher.subscribe(async (c: CipherView) => {
+        childComponent.onRestoredCipher.subscribe(async () => {
             this.modal.close();
```

Comment:
Why was this changed? The signature for these outputs passes a CipherView:

```
    @Output() onSavedCipher = new EventEmitter<CipherView>();
    @Output() onDeletedCipher = new EventEmitter<CipherView>();
    @Output() onRestoredCipher = new EventEmitter<CipherView>();
    @Output() onCancelled = new EventEmitter<CipherView>();
    @Output() onEditAttachments = new EventEmitter<CipherView>();
    @Output() onShareCipher = new EventEmitter<CipherView>();
    @Output() onEditCollections = new EventEmitter<CipherView>();

```

Reply:
This was just a linter recommendation, since the value is never used, but I could see a reason for keeping it around for context.

Reply:
Let me know if you still want this put back

Reply:
Please do, thanks @addisonbeck

Reply:
It is done 😁